### DISCUSSION TITLE
Guess the artist 🖼️🎨🖌️

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1267] [Galaxy Trivia] [answered]
![image](https://github.com/ParanoidUser/yolo/assets/5120290/19059546-f265-403b-a0b1-679bf2accf97)

**Comment by tomerf** [comment]
Frida Kahlo

**Comment by p3rception** [comment, marked as answer]
The artist is Frida Kahlo!
